LessOOP/Program.cs

```diff
@@ -1,48 +1,96 @@
+            // Demo5();//работа с объектом рациональных чисел
+            // Demo6();//работа в объекте счет с операторами сравнения
+               Demo7();//фигуры
+        }
+
+        static void Demo7()
+        {
+            var c = new Circle(0,0,10);//создаем объект круг
+                c.MoveHorisontal(5);//движение по горизонтали
+                c.MoveVertical(7);//движение по вертикали
+            var d=c.Square();//площадь
+                 c.Print();//вывод на консоль параметров объекта
+
+            var rec = new Rectangle(0, 0, 15, 10);
+            rec.Print();//вывод на консоль параметров объекта
+        }
+
+        static void Demo6()
+        {
+            var acc1 = new Accounts(1, 100, (typeacc)1);
+            var acc3 = new Accounts(1, 100, (typeacc)2);
+            var acc2= acc1;
+
+            var h = acc1.Equals(acc2);
+            var h1 = Equals(acc1,acc2);
+            var h2 = Equals(acc1, acc3);
+
+        }
+
+            static void Demo5()
+        {
+            var rn1 = new RationalNumber(1,5);
+            var rn2 = new RationalNumber(2, 5);
+
+            var rn3 = rn1 + rn2;
+            var rn4 = rn1 - rn2;
+            var rn5 = rn1 * rn2;
+            var rn6 = rn1 / rn2;
+            var b = rn1 == rn2;
+            var b1 = rn1 != rn2;
+            var b2 = rn1 > rn2;
+            var b3 = rn1 < rn2;
+
+            var rn7 = (RationalNumber)"7/8";
+
+        }
+
+        static void Demo1()
+        {
+            Accounts[] Arr = new Accounts[4];//массив счетов
             //Создаем четыре счета
-            Arr[0] = new Accounts(1, 100, (typeacc) 1);
+            Arr[0] = new Accounts(1, 100, (typeacc)1);
             Arr[1] = new Accounts(1, 50, (typeacc)2);
             Arr[2] = new Accounts(1, 400, (typeacc)1);
             Arr[3] = new Accounts(1, 70, (typeacc)1);
 
-            Accounts acc = Arr[0];//текущий счет
+            //Accounts acc = Arr[0];//текущий счет
+            Accounts acc;//текущий счет
 
             Console.WriteLine($"Доступные счета");
-            foreach (Accounts accoun in Arr) 
+            foreach (Accounts accoun in Arr)
             {
                 Console.WriteLine($"Номер счета:{accoun.NumberAccounts}  На балансе сумма: {accoun.BalansAccounts} Тип счета:{accoun.TypeAccounts}");
             }
 
             Console.WriteLine($"Введите номер счета");
             var intextNum = Console.ReadLine();
-            long  num = Convert.ToInt64(intextNum);
-            Boolean flag = false;
-           
-            //Ищем выбранный счет
-            foreach (Accounts accnt in Arr)
-            {
-                if (num == accnt.NumberAccounts) 
-                {
-                    acc = accnt;
-                    flag = true;
-                }  
-                if (flag == true) break;
-            }
+            long num = Convert.ToInt64(intextNum);
 
-            if (flag == true)
+            if (FindeAcc(num, Arr, out acc) == true)
```

Comment:
Индусский код! Не пишите так.
```suggestion
            if (FindeAcc(num, Arr, out acc))
```
Результат вызова функции FindeAcc - это уже значение bool.